pyani/fastani.py

```diff
@@ -92,9 +92,13 @@ def get_version(fastani_exe: Path = pyani_config.FASTANI_DEFAULT) -> str:
     """
     try:
         fastani_path = Path(shutil.which(fastani_exe))  # type:ignore
+    # Returns a TypeError if `fastani_exe` is None
     except TypeError:
-        return f"{fastani_exe} is not found in $PATH"
+        return f"expected file location; received {fastani_exe}"
```

Comment:
"file location" maybe isn't specific enough, here. I propose changing to "path to fastANI executable"